src/Form/SpeakerType.php

```diff
@@ -45,7 +45,7 @@ public function buildForm(FormBuilderInterface $builder, array $options): void
             ->add(
                 'photo',
                 FileType::class,
-                ['required' => false]
+                ['required' => true]
```

Comment:
Missing trailing comma

Reply:
If I do that my work with the default image will not help.

Reply:
if I put my work on the photo by default will not help.

Reply:
if I add a final comma behind ['required' => false] my editor generates an error